torch_frame/nn/encoding/positional_encoding.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import torch
+from torch import Tensor
+from torch.nn import Module
+
+
+class PositionalEncoding(Module):
+    r"""Positional encoding introduced in `"Attention Is All You Need"
+    <https://arxiv.org/abs/1706.03762>`_ paper. Given an input tensor of shape
+    :obj:`(*, )`, this encoding expands it into an output tensor of shape
+    :obj:`(*, out_size)`.
+
+    Args:
+        out_size (int): The output dimension size.
+    """
+    def __init__(self, out_size: int):
+        super().__init__()
+        if out_size % 2 != 0:
+            raise ValueError(
+                f"out_size should be divisible by 2 (got {out_size}).")
+        self.out_size = out_size
+        mult_term = torch.exp(-np.log(10000.0) *
```

Comment:
Why we use np.log here. Change to torch?

Reply:
oops